src/config/mgdLDefaults.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
[nit] Consider putting these in a `defaults.js` and have, at the top-level, a `mgdL` dictionary and a `mmolL` dictionary. Then you could import it as just `defaults` and use it as `defaults.mgdL`.